prog/postgres/postgres_server_nexus.rb

```diff
@@ -166,10 +166,41 @@ def before_run
 CONFIG
     vm.sshable.cmd("sudo -u prometheus tee /home/prometheus/web-config.yml > /dev/null", stdin: web_config)
 
-    vm.sshable.cmd("sudo systemctl enable --now postgres_exporter")
-    vm.sshable.cmd("sudo systemctl enable --now node_exporter")
-    vm.sshable.cmd("sudo systemctl enable --now prometheus")
-    hop_configure
+    metric_destinations = postgres_server.resource.metric_destinations.map {
+      <<METRIC_DESTINATION
+- url: '#{_1.url}'
+  basic_auth:
+    username: '#{_1.username}'
+    password: '#{_1.password}'
+METRIC_DESTINATION
+    }.prepend("remote_write:").join("\n")
+
+    prometheus_config = <<CONFIG
+global:
```

Comment:
The default retention period is 15 days, with no limit on size. Do we have an estimate for disk usage? Since we're writing data remotely and no one will directly connect to this prometheus server and query metrics, should we consider reducing the retention time aggressively?

https://prometheus.io/docs/prometheus/latest/storage/#operational-aspects

Reply:
Based on my rought, back of the napkin calculations, 15 days retention would require ~1GB data. I'm also planning to reduce the number of metrics we emit, because by default prometheus emit lots of metrics that are not useful in our context. That would reduce the overall size further.

I added storage limit (to the image generation PR).